packages/server/src/controllers/UsersController.js

```diff
@@ -4,7 +4,145 @@ import UsersValidator from '../validators/UsersValidator'
 
 const { User, UserAddress } = Models
 
+// temp dummy models
+let Users = { }
+let UserEmails = { }
+let UserSessions = { }
+
+
 export default {
+  async details(req, res) {
+    const { params: { address } } = req
+    let body = {
+      emailExists: false,
+      emailVerified: false,
+      addressVerified: false,
+      notificationsDisabled: false
+    }
+    Object.assign(body, Users[address])  // Object.assign acts as a merge to avoid empty keys
+    res.send(body)
+  },
+
+
+  sessions: {
+    async create(req, res) {
+      const { params: { address } } = req
+      if (!(address in Users)) {
```

Comment:
_Usually_ style guides will prefer you to avoid `in` because it has loose semantics (more verbose than just property access, less specific than just `hasOwnProperty()`).

This is a bit of a nitpick now because this operation would involve a DB query later and will get replaced, but just in case we keep some of the code for later :).